### DISCUSSION TITLE
SCSS support in Vue files

### DISCUSSION THREAD

**Discussion by julian-weimer** [helix-editor/helix#10531] [Troubleshooting]
I noticed that i get no syntax highlighting when i use `lang="scss"` in Vue files

<img width="240" alt="Bildschirmfoto 2024-04-20 um 11 20 49" src="https://github.com/helix-editor/helix/assets/44260676/ea69752b-563c-46a4-842f-73a8a516ba8b">
 
<img width="248" alt="Bildschirmfoto 2024-04-20 um 11 20 57" src="https://github.com/helix-editor/helix/assets/44260676/43daf69d-56e0-496b-85c1-ba7b89a34929">

Current Vue configuration in `languages.toml`:

```
[[language]]
name = "vue"
auto-format = true
formatter = { command = 'npx', args = ["prettier", "--parser", "vue"] }
language-servers = ["typescript-language-server", "vscode-css-language-server"]
scope = "text.html.vue"
roots = ["package.json"]
injection-regex = "vue"

[[language-server.typescript-language-server.config.plugins]]
name = "@vue/typescript-plugin"
location = "[...my node_modules location]/@vue/language-server/"
languages = ["vue"]
```
I don't know if it's really needed, but i added the `vscode-css-language-server` to `language-servers`.

Using helix 24.3 (2cadec0b)

Is there anything that i can configure?

**Comment by the-mikedavis** [comment]
I can't reproduce this - a block of

```vue
<style lang="scss">
body {
  color: red;
}
</style>
```

in a Vue file shows SCSS highlighting for me. Do you have any other config in your `languages.toml`? Also which theme are you using?

**Reply by julian-weimer** [reply]
Hi @the-mikedavis,

I commented out all my existing configuration in my `languages.toml` and repeated the test using the snippet above. HTML files seem to work fine, but files with a `.vue` extension not.

I'm using the `catppuccin_mocha` theme.

Strange...
